# ecss/no-extend

ECSS recommends avoid `@extend`.

## GOODğŸ‘

```scss
.foo {
  color: #000;
}

.bar {
  color: #fff;
}
```

## BADğŸ‘

```scss
.foo {
  color: #000;
}

.bar {
  @extend .foo;
  color: #fff;
}
```
